.github/workflows/main.yml

```
@@ -56,14 +56,19 @@ jobs:
           name: python-artifacts
           path: .
 
+      - name: Upload helm artifacts
+        uses: actions/upload-artifact@v4
+        with:
+          name: helm-artifacts
+          path: ${{ github.workspace }}/helm-charts
+
   docker-ci:
     runs-on: ubuntu-latest
     if: ${{ github.ref_name == 'main' }}
```

For pull requests from a fork, the branch from which the PR is created has value of github.ref_name not equal main  branch, so this condition is not met and actions is skipped
```suggestion
    if: ${{ github.base_ref == 'main' }}
```